benchmarks/src/main/scala/zio/ChunkBenchmarks.scala

```diff
@@ -0,0 +1,53 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import zio._
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class ChunkBenchmarks {
+
+  @Param(Array("1000"))
```

Comment:
It might be interesting to verify whether performance is affected by size change.